javatests/arcs/android/host/AndroidEntityHandleManagerTest.kt

```diff
@@ -41,7 +45,14 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import kotlin.coroutines.experimental.suspendCoroutine
 
+// Generated from ./javatests/arcs/android/host/test.arcs
 private typealias Person = TestParticleInternal1
+private typealias PersonWithQuery = TestParticleInternal2
+
+fun Person.withQuery(): PersonWithQuery {
+    // TODO(cypher1): Remove this when implicit casting to super types is supported by code gen.
```

Comment:
Might be best to use a buganizer bug in the todo's parens, rather than your username - so we can track it.